client/src/components/CarProps.js

```diff
@@ -1,24 +1,135 @@
-export default function CarProps({userData, handleChange}) {
-    if (userData.car) {
-        return (
-            <>
-            <label htmlFor="carAge">Alter des Fahrzeugs:</label>
-            <input
-              type="text"
-              name="carAge"
-              value={userData.carAge}
-              onChange={handleChange}
-            />
-            <label htmlFor="carValue">Wert des Fahrzeugs:</label>
-            <input
-              type="text"
-              name="carValue"
-              value={userData.carValue}
-              onChange={handleChange}
-            />
-            </>
-        )
-    } else {
-        return ''
-    }
-}
\ No newline at end of file
+import styled from "styled-components";
+
+export default function CarProps({
+  userData,
+  handleChange,
+  focused,
+  handleClick,
+  formValidation,
+}) {
+  if (userData.car) {
```

Comment:
I would recommend, instead of having the condition within this component to have this outside (where you render the component) `{userData.car && <CarProps userData={userData} .... />}`